scripts/build_proxies.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -o nounset errexit pipefail
+
+# Collect the API Proxies and Hosted Targets
+# files into dist/proxies/
+
+rm -rf dist/proxies
+mkdir -p dist/proxies/sandbox
+mkdir -p dist/proxies/live
+cp -Rv proxies/sandbox/apiproxy dist/proxies/sandbox
+cp -Rv proxies/live/apiproxy dist/proxies/live
+mkdir -p dist/proxies/sandbox/apiproxy/resources/hosted
+mkdir -p dist/proxies/live/apiproxy/resources/hosted
+rsync -av --copy-links --exclude="node_modules" --filter=':- .gitignore' sandbox/ dist/proxies/sandbox/apiproxy/resources/hosted
+rsync -av --copy-links --filter=':- .gitignore' coordinator/dist/ dist/proxies/live/apiproxy/resources/hosted
+cp coordinator/package.json dist/proxies/live/apiproxy/resources/hosted
+cp coordinator/app.yaml dist/proxies/live/apiproxy/resources/hosted
```

Comment:
The other build commands are in the makefile, this is the only one in a seperate script. Should we move the others into scripts? Or should this be made part of the makefile?

Reply:
Think that's worth reviewing once we've used the new format for models for a while and see if we're happy with those changes. The scripts folder in general might be one to consider refactoring to help see where they are used. This script has been here a while, I just renamed it from build_proxy to build_proxies